src/assessment.py

```diff
@@ -47,6 +48,8 @@ def post_assessment():
         return str(e), 400
     except InvalidResponseError as e:
         return f'InvalidResponseError: {str(e)}', 400
+    except KeyConceptError as e:
+        return e, 400
```

Comment:
The details are recorded when the error is raised, so returning e displays those details to the user.

Reply:
Does it just implicitly convert the error to `str`? That's nice and clean, if so, and I wonder if we should do that in all of the error cases instead of doing three different things. 😂

Reply:
I think the way that you do it in the InvalidResponseError is fine too. I believe you need to do the string conversion to be able to add it to an fstring, but not if you just pass the error message from where it was raised.

Reply:
I prepended InvalidResponseError because I wanted to make sure the error name showed up as part of what was returned. I'd prefer that information not be removed from the previous clause, since I think it is helpful to be able to see at a glance that the problem was bad data returned by OpenAI. My personal choice would have been to include the exception for KeyConceptError as well, but I didn't feel strongly enough about it to request it in code review 😅